### ISSUE TITLE
Timers page incorrect timers after midnight

### ISSUE THREAD

**Issue by darknite323** [gw2efficiency/issues#1710]
Got what seems like a timezone related bug on the Timers page.
if my local time has gone past midnight, the page just starts showing the next set of events to start in something like 8 hours or more.
but if i change my PC's timezone to one where its before midnight, the timers show just fine.
I have triple checked my PC's settings, my local timezone is set correctly and automatically updates the time.

I have also tested this in multiple web browsers, and in incognito modes. not logged in, with no API key.

examples:
Local time is 1:48am (UTC+10), but this is what shows on the site:
![image](https://github.com/gw2efficiency/issues/assets/49840343/7d645f79-046a-4e89-a532-05ed33ca47bc)

However if i manually change my local timezone to UTC+8 (which makes Windows show the time as 11:49am now), this shows:
![image](https://github.com/gw2efficiency/issues/assets/49840343/922cb743-84a1-4ab8-9a78-fc92dbbc1b63)

Which is now reporting accurate times that match the wiki event timers page.

**Comment by queicherius** [comment]
Just to double check - during this your personal system's time was correct? (So far all reports I had about this ended up being the system time being off, so worth the quick look.)

**Comment by darknite323** [comment]
> Just to double check - during this your personal system's time was correct? (So far all reports I had about this ended up being the system time being off, so worth the quick look.)

Yep i can't find anything wrong with the local time, I can have it "sync now", and it syncs successfully, I can change it to another server to sync with, but that doesn't change the time at all, and timezone is set correctly.
I'd understand it being off if say, my local timezone was set incorrectly and the computer time was set manually to compensate.
 
![image](https://github.com/gw2efficiency/issues/assets/49840343/0c92d7df-de5e-44fd-bde5-53860be125fc)

**Comment by queicherius** [comment]
👍 Alright, I'll have a look to see what's going on soon.
